test/integration/targets/nxos_install_os/tasks/upgrade/copy_kick_system_images.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: "Setup - Turn on feature scp-server"
+  nxos_feature:
+    feature: scp-server
+    state: enabled
+
+- name: "Copy {{ si }} to bootflash"
+  nxos_file_copy:
+    local_file: "{{image_dir}}{{ si }}"
+    file_system: "bootflash:"
+    host: "{{ inventory_hostname }}"
```

Comment:
`host` shouldn't be required. Also with the latest commit https://github.com/ansible/ansible/commit/c2f7f36fc5a476a8f16a8d9c47afbc4e24f696c6 running this with nxapi will fail.